tst/testinstall/float.tst

```diff
@@ -179,12 +179,12 @@ gap> 1.5e10;
 gap> -1.5e0;
 -1.5
 gap> 0.7e-10;
-7.e-11
+7e-11.
```

Comment:
Hmmm, that's not a valid floating point :/. 
```
gap> 7e14;
Error, Variable: '7e14' must have a value
not in any function at *stdin*:14
gap> 7e14.;
Error, Variable: '7e14' must have a value
not in any function at *stdin*:15
Syntax error: Record component name expected
7e14.;
     ^
gap> 7.e14;
7.e+14
```

I guess that means the logic in your patch needs to be a bit more complicated. I guess looking at `MACFLOAT_STRING_DOTTIFY` to see what it does, and then adapt it in the C code, would do the trick.